.env

```diff
@@ -0,0 +1 @@
+SASS_PATH=./src/assets/styles
```

Comment:
`.env` file shouldn't be pushed to GitHub

Reply:
Actually it should if following CRA: https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env

Reply:
They should, according to create-react-app docs :D https://create-react-app.dev/docs/adding-custom-environment-variables#docsNav

Reply:
Fair enough